README.md

```diff
@@ -121,7 +134,7 @@ for i, ax in enumerate(axs):
     pdep, confi = gam.partial_dependence(XX, feature=i+1, width=.95)
 
     ax.plot(XX[:, i], pdep)
-    ax.plot(XX[:, i], confi, c='r', ls='--')
+    ax.plot(XX[:, i], confi[0], c='r', ls='--')
```

Comment:
`confi` is a list containing a 1D numpy array. I had to get that array to get this line to work; otherwise I got `ValueError: x and y must be the same size`

Reply:
awesome!!